test/mongooseim_metrics_SUITE.erl

```diff
@@ -91,6 +105,15 @@ tcp_connections_detected(_C) ->
     {ok, [{value, X}]} = mongoose_metrics:get_metric_value(global, tcpPortsUsed),
     ?assert(X > 0).
 
+queued_messages_increase(_C) ->
+    Pids = [spawn(fun() -> receive die -> ok end end) || _ <- lists:seq(1,5)],
+    lists:foreach(fun(Pid) -> Pid ! undefined end, Pids),
+    {ok, {ok, _L}} = async_helper:wait_until(
+                fun() ->  mongoose_metrics:get_metric_value(global, processQueueLengths) end,
+                {ok, [{fsm, 0}, {regular, 5}, {total, 5}]}
```

Comment:
1. I'm not sure if we should match on the exact order of the fields. I'd apply `lists:sort` on the expected and returned value.
2. It's easy to validate `fsm` value as well - spawned processes have to simply set `$internal_queue_len` dictionary value to something higher than 0.